CCGlucose/Classes/Glucose.swift

```diff
@@ -192,8 +192,14 @@ public class Glucose : NSObject {
     }
     
     func parseGlucoseMeasurement(data:NSData) {
-        let glucoseMeasurement = GlucoseMeasurement(data: data)
-        glucoseDelegate?.glucoseMeasurement(measurement: glucoseMeasurement)
+        // ensure the first byte is not zero before parsing the data
+        var values = [UInt8](repeating:0, count: 1)
+        data.getBytes(&values, length: 1)
+        
+        if (values[0] != 0) {
```

Comment:
NSData is bridged to Data, so you can just do:
```
let nsdata = data as Data
if nsdata[0] != 0 {
}
```
Also, no need for parenthesis on if statements.